cmus.c

```diff
@@ -104,6 +105,22 @@ void cmus_prev(void)
 		player_set_file(info);
 }
 
+void cmus_rand(void)
+{
+	struct track_info *info;
+
+	view_clear(QUEUE_VIEW);
```

Comment:
Is it okay to clear the queue this way? `cmus_next()` has thread synchronization when removing items from the queue, but I didn't dig into why.

Reply:
I think so, as long as `cmus_rand` is only being called from the UI thread.